GeoLib/AABB.h

```diff
@@ -85,6 +85,20 @@ class AABB
 		}
 	}
 
+	/**
+	 * Construction of object using input pointer array.
+	 * @param pnt_array  an array of point pointers
+	 * @param size       size of the array
+	 */
+	AABB(PNT_TYPE * const* pnt_array, unsigned size) :
```

Comment:
we cannot write `ele.getNodes().begin()` since `Element::getNodes()` returns `Node**`. 

I also thought about `std::begin()` and `std::end()` but they can be used only for static array (e.g. int a[4]) as `std::end()` internally calls sizeof(). Is it right?